apps/prairielearn/elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -224,10 +226,17 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     formula_editor = pl.get_boolean_attrib(
         element, "formula-editor", SHOW_FORMULA_EDITOR_DEFAULT
     )
+    initial_value = pl.get_string_attrib(
+        element, "initial-value", INITIAL_VALUE_DEFAULT
+    )
     raw_submitted_answer_latex = data["raw_submitted_answers"].get(
         name + "-latex", None
     )
     raw_submitted_answer = data["raw_submitted_answers"].get(name, None)
+    if raw_submitted_answer is None:
+        raw_submitted_answer = initial_value
+    if raw_submitted_answer_latex is None:
+        raw_submitted_answer_latex = initial_value
```

Comment:
I'm not convinced this will work as expected. Have you tested this with and without the formula editor?